### ISSUE TITLE
Find not working after upgrade from 3.4.0.0 to 3.5.3.0 

### ISSUE THREAD

**Issue by LennyHoglund** [FortnoxAB/csharp-api-sdk#127]
Hello Richard,

Just upgraded from API version 3.4.0.0 to 3.5.3.0 and are getting into problems with the .Find() and .Find(searchSettings) statements. Maybe the problem is related to the issue described by bpaola71. Anyway, here are some more information:

In Find() the search goes on forever.
In Find(searchSettings) we receive the compile error: CS1501: No overload for method 'Find' takes 1 arguments.
All other Fortnox API calls works fine in our projects but the .Find() problem is currently out off our control.

By the way, we used "Connector.Page = 1" in 3.4.0.0 to limit the number of rows received in each .Find(). How is that achieved in 3.5.3.0? 

Very grateful for for any suggestion on how to solve the problem. 
Thanks for making the Fortnox API better and better.

Best regards,
Lennart Hoglund

 P.S. Tested FinancialYearConnector.cs in your FortnoxAPILibrary.Tests project and it worked fine. D.S.

**Comment by richardrandak** [comment]
Hello Lenny!
I am little bit confused from the question.
The v3.5.3 does not have searchSettings arguments. It is working as before, defining search parameters as connector's properties.
It is possible you are using the "beta" version where the search was reworked. In that case, you should always inject the search params object into Find method. Every search object has Page property to define which page you want.
Basically, should work the same as before, except instead using connector's properties, you create a XyzSearch object and inject it to Find method. 

In which connector did this error occur? `No overload for method 'Find' takes 1 arguments.`

**Comment by LennyHoglund** [comment]
Hello Richard,

In the attached documents you can see how we use the Find statement. In version 3.3.2.0 it worked fine but when we upgraded to 3.5.3.0 the connector does not accept the Page method.
I am sure we are missing something in our code related to Page, but we just can’t figure out what. We have tried - Connector.Search.Page = 1; - but then we run into a loop when - var Currencies = Connector.Find(); - is executed.

If you could give us a hint about how to inject the search params object into Find method for 3.5.3.0 that will make us very happy.

Thank you very much for your reply. Very appreciated,

Lennart Hoglund

[cid:image001.png@01D6E2BB.2DA52E10]
Lennart Hoglund
CEO & Senior System Architect

Lewab AB
Nytorgsgatan 24
116 22  Stockholm
Sweden
Phone    +46 (0) 8 641 3911
Mobile   +46 (0)70 646 1328
E-mail     lennart.hoglund@lewab.com<mailto:lennart.hoglund@lewab.com>
Site          www.lewab.com

**Comment by richardrandak** [comment]
I don't see the attachments.. could you send it directly to my mail `richard.randak@softwerk.se` ?

**Comment by LennyHoglund** [comment]
Roger that …

[cid:image001.png@01D6E357.852F2E50]
Lennart Hoglund
CEO & Senior System Architect

Lewab AB
Nytorgsgatan 24
116 22  Stockholm
Sweden
Phone    +46 (0) 8 641 3911
Mobile   +46 (0)70 646 1328
E-mail     lennart.hoglund@lewab.com<mailto:lennart.hoglund@lewab.com>
Site          www.lewab.com

**Comment by richardrandak** [comment]
Issue was caused by a missing "ConfigureAwait(false)" statement in the underlying implementation of Find method.
A new version v3.5.4 with hotfix was released as NuGet.
